NEWS.md

```diff
@@ -1,7 +1,7 @@
 # gtrendsR (development version)
 
 # gtrendsR 1.4.4
-
+- Added the possibility to use search operators like "+" and "-"
```

Comment:
This is not placed in the right section. 1.4.4 is already published.

Reply:
I have added it to the gtrends development version